tests/index/hnswlib/test_persist_data.py

```diff
@@ -81,5 +81,7 @@ def test_persist_and_restore_nested(tmp_path):
 
 
 def test_persist_index_file(tmp_path):
-    _ = HnswDocumentIndex[SimpleDoc](work_dir=str(tmp_path))
-    _ = HnswDocumentIndex[SimpleDoc](work_dir=str(tmp_path))
+    store = HnswDocumentIndex[SimpleDoc](work_dir=str(tmp_path))
```

Comment:
can u call them `index` better, to have a clear distinction between indexes and stores

Reply:
> can u call them `index` better, to have a clear distinction between indexes and stores

`store` is used in all existing test files. Can I change them all in this PR? @JoanFM

Reply:
yes